docker_cli/_docker_cmake.py

```diff
@@ -0,0 +1,59 @@
+from textwrap import dedent
+
+from ._docker_mamba import micromamba_docker_lines
+from .defaults import build_prefix, install_prefix, src_prefix
+
+
+def cmake_config_dockerfile(base: str, build_type: str, with_cuda: bool = True) -> str:
+    """
+    Creates a Dockerfile for configuring CMake Build.
```

Comment:
Let's add a note that mentions that we assume that the current working directory is the source directory (that contains the top-level `CMakeLists.txt` file).